.circleci/config.yml

```diff
@@ -4,55 +4,111 @@
 version: 2
 jobs:
   build:
+    working_directory: ~/alice
     docker:
       - image: circleci/elixir:1.7
         environment:
           MIX_ENV: test
-
-    working_directory: ~/alice
     steps:
       - checkout
+      # - run:
+      #     name: "Creating master branch cache key"
+      #     command: git log origin/master --pretty=format:'%H' -n 1 > latest-master-sha
+      # - run:
+      #     name: "Creating build branch cache key"
+      #     command: git log --pretty=format:'%H' -n 1 > build-branch-sha
       - run:
-          name: "Creating master branch cache key"
-          command: git log origin/master --pretty=format:'%H' -n 1 > lastest-master-sha
-      - run:
-          name: "Creating build branch cache key"
-          command: git log --pretty=format:'%H' -n 1 > build-branch-sha
-      - run: mix local.hex --force
-      - run: mix local.rebar --force
-      - restore_cache: # restores saved mix cache
+          name: "Setting up hex and rebar"
+          command: |
+            mix local.hex --force
+            mix local.rebar --force
+      - restore_cache:
           name: "Restore dependency cache"
           keys:
             - v1-alice-dependency-cache-{{ .Branch }}-{{ checksum "mix.lock" }}
             - v1-alice-dependency-cache-{{ .Branch }}
             - v1-alice-dependency-cache-master-{{ checksum "mix.lock" }}
             - v1-alice-dependency-cache-master
-      - restore_cache:
-          name: "Restore Build & PLT cache"
-          keys:
-            - v1-alice-build-and-plt-cache-{{ .Branch }}-{{ checksum "build-branch-sha" }}
-            - v1-alice-build-and-plt-cache-{{ .Branch }}
-            - v1-alice-build-and-plt-cache-master-{{ checksum "lastest-master-sha" }}
-            - v1-alice-build-and-plt-cache-master
-      - run: mix do deps.get, compile  # get updated dependencies & compile them
-      - run: mix compile --force --warnings-as-errors # compile project and treat warnings as error
+      # - restore_cache:
+      #     name: "Restore Build & PLT cache"
+      #     keys:
+      #       - v1-alice-build-and-plt-cache-{{ .Branch }}-{{ checksum "build-branch-sha" }}
+      #       - v1-alice-build-and-plt-cache-{{ .Branch }}
+      #       - v1-alice-build-and-plt-cache-master-{{ checksum "latest-master-sha" }}
+      #       - v1-alice-build-and-plt-cache-master
```

Comment:
This should likely be deleted rather than commented out.

Reply:
Yeah I want to do that before merging this, but before that, I need to make a branch this this stuff in it